projects/minesweeper/src/main/java/org/batfish/minesweeper/bdd/TransferBDD.java

```diff
@@ -972,6 +976,10 @@ BDDRoute ite(BDD guard, BDDRoute r1, BDDRoute r2) {
 
     ret.setUnsupported(ite(guard, r1.getUnsupported(), r2.getUnsupported()));
 
+    MutableBDDInteger i =
```

Comment:
This was missing for setOspfMetric to work properly (other cases too).
I believe this is not on the code path for `CompareRoutingPolicies`, only SRP, hence why my unit tests for comparison did not uncover the bug.
The unit test added for `TransferBDD` caught this.